src/vs/base/browser/ui/selectBox/selectBox.css

```diff
@@ -6,4 +6,63 @@
 .monaco-workbench .select-box {
 	width: 100%;
 	height: 20px;
-}
\ No newline at end of file
+}
+
+/* Require .monaco-shell for ContextView dropdown */
+
+.monaco-shell .select-dropdown-container  {
+	display: none;
+}
+
+.monaco-shell .select-dropdown-container.visible  {
+	display: flex;
+	flex-direction: column;
+	text-align: left;
+	width: 1px;
+	overflow: hidden !important;
+}
+
+.monaco-shell .select-dropdown-container .selectbox-dropdown-list-container {
```

Comment:
@cleidigh I think a hierarchy should be expressed in CSS if clearly present. E.g. `.select-dropdown-container .selectbox-dropdown-list-container` can be `.select-dropdown-container > .selectbox-dropdown-list-container` imho

Reply:
Done.